src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
+
+  return totalCost;
```

Comment:
The function name `calculateRentalCost` is descriptive and clearly indicates its purpose. However, the implementation uses hardcoded literals for the discount values and the minimum days required for discounts, which should be replaced with well-named constants to improve code readability and maintainability.